### DISCUSSION TITLE
Welcome to tinyproxy Discussions!

### DISCUSSION THREAD

**Discussion by obnoxxx** [tinyproxy/tinyproxy#472] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by obnoxxx** [comment]
Note, this was just the first topic auto-createwd  when I enabled discussions ... - @rofl0r - fyi.

**Comment by anu743** [comment]
i am facing one issue with tinyproxy ,
for every connection tinyproxy will take 2 MB memory consumption and CPU load , after the request completes ,does this memory released,
when i try top -H command and check the VIRT - virutal memory consumed by the tinyproxy is not released over the period , even the request has been completed.

and i tried with valgrind also but i haven't found any leaks as well.

can someone please clarify me the memory management inside the tinyproxy?

**Reply by rofl0r** [reply]
@anu743 please open a new issue and fill in the required information such as tinyproxy version. this topic here is for introduction, not for issue discussion.
